docs: cannabilize sphinx documentation into main README (#355)
Unleash/unleash-client-python: README.md

Unleash is a private, secure, and scalable [feature management platform](https://www.getunleash.io/) built to reduce the risk of releasing new features and accelerate software development. This server-side Python SDK is designed to help you integrate with Unleash and evaluate feature flags inside your application.

You can use this client with [Unleash Enterprise](https://www.getunleash.io/pricing?utm_source=readme&utm_medium=python) or [Unleash Open Source](https://github.com/Unleash/unleash).

>  **Migrating to v6**
>
> If you use custom strategies or access the `features` property on the Unleash Client, read the complete [migration guide](./v6_MIGRATION_GUIDE.md) before upgrading to v6.

## Getting Started

### Install the Unleash Client in your project

```bash
pip install UnleashClient
```

### Initialization

> You must initialize the SDK before you use it. Note that until the SDK has synchronized with the API, all features will evaluate to `false` unless

Author: There's no way to force a sync in the Python SDK

you have a [bootstrapped configuration](#bootstrap) or you use [fallbacks](#fallback-function).

```python
from UnleashClient import UnleashClient

client = UnleashClient(

>     url="https://YOUR-API-URL",

Reviewer: Just for consistency with the other placeholder
```suggestion
    url="https:<YOUR-API-URL>",
```

A custom strategy is just a class that implements an apply method.

``` python

class ActiveForEmailStrategy:
    def apply(self, parameters: dict, context: dict = None) -> bool:
        # Decide if the feature is active for this context
        return context.get("email") in parameters

```

Once you’ve defined your strategy, register it when you initialize the client. The key must match the strategy name in Unleash exactly.

``` python
## You should have a custom strategy defined in Unleash called 'EmailStrategy'
my_custom_strategies = {
    "EmailStrategy": ActiveForEmailStrategy()
}

client = UnleashClient(

    url="https://YOUR-API-URL",
    app_name="my-python-app",
    custom_headers={'Authorization': '<API token>'},
    custom_strategies = my_custom_strategies
)

```

### Impression data

The Python SDK supports [impression data](https://docs.getunleash.io/reference/impression-data). This lets you capture an event for every feature toggle evaluation.

**Note**: The SDK does not include a built-in event bus — you’ll need to provide your own. The example below shows how to use [Blinker](https://pypi.org/project/blinker/) to send signals.

To use impression data:

> - Enable Impression Data on your feature flags in the Unleash UI.

Reviewer: ```suggestion
- Enable impression data on your feature flags in the Unleash UI.
```

Your callback must accept a single UnleashEvent. You can log it, store it, or send it to another system.

```python
from blinker import signal
from UnleashClient import UnleashClient
from UnleashClient.events import UnleashEvent

send_data = signal('send-data')

@send_data.connect
def receive_data(sender, **kw):
    print("Caught signal from %r, data %r" % (sender, kw))

def example_callback(event: UnleashEvent):
    send_data.send('anonymous', data=event)

client = UnleashClient(
    url="https://YOUR-API-URL",
    app_name="my-python-app",
    custom_headers={'Authorization': '<API token>'},
    event_callback=example_callback
)
client.initialize_client()
client.is_enabled("testFlag")

```

Impression callbacks run in-process — keep them fast to avoid blocking your app.

### Custom cache

By default, the Python SDK stores feature toggles in an on-disk cache using fcache. If you need a different storage backend, for example, Redis, memory-only, or a custom database, you can provide your own cache implementation.

Below is an example custom CustomCache using fcache under the hood.

```python
from typing import Optional, Any
from UnleashClient.cache import BaseCache

> from fcache.cache import FileCache as _Filecache

Reviewer: ```suggestion
from fcache.cache import FileCache as _FileCache
```

    def set(self, key: str, value: Any):
        self._cache[key] = value
        self._cache.sync()

    def mset(self, data: dict):
        self._cache.update(data)
        self._cache.sync()

    def get(self, key: str, default: Optional[Any] = None):
        return self._cache.get(key, default)

    def exists(self, key: str):
        return key in self._cache

    def destroy(self):
        return self._cache.delete()
```
Pass your cache instance to the client with the cache argument:

```python
client = UnleashClient(
    url="https://YOUR-API-URL",
    app_name="my-python-app",
    cache=CustomCache("my-cache")
)
```

## Running in multi-process setups

The Python SDK runs a background thread to keep feature toggles in sync with the Unleash server. Some runtime environments, like WSGI servers and Celery workers, need extra setup to make sure the SDK works correctly.

### WSGI

When using WSGI servers (e.g., for Flask or Django apps), be aware that:

- Many WSGI setups disable threading by default. The SDK needs threads to poll for updates in the background.
- Make sure to set `enable-threads` in your WSGI config.

> If you use multiple processes (processes flag) for scaling, you may need to enable the `lazy-apps` flag. This ensures each process gets a fresh SDK instance.

Reviewer: Would this work? Flags confuses me, even though it might be right word
```suggestion
When running under uWSGI with multiple processes (using the `--processes` option), you may need to enable the `lazy-apps` option. This ensures each process gets a fresh SDK instance.
```

Author: Yeah fair, I think that works nicely

See [The Art of Graceful Reloading](https://uwsgi-docs.readthedocs.io/en/latest/articles/TheArtOfGracefulReloading.html#preforking-vs-lazy-apps-vs-lazy) for more details.

### Celery

When using the SDK in Celery tasks, make sure you initialize it inside the worker_process_init event. Otherwise, the worker may run but won’t poll for feature toggle updates.

```python
from UnleashClient import UnleashClient
from celery.signals import worker_process_init

client = UnleashClient(
    url="https://YOUR-API-URL",
    app_name="my-python-app",
    custom_headers={'Authorization': '<API token>'}
)

@worker_process_init.connect
def configure_workers(sender=None, conf=None, **kwargs):
    client.initialize_client()
```

> ## Contributing & Development

Reviewer: ```suggestion
## Contributing and development
```
